### ISSUE TITLE
Vim9: wrong error line number when a :def function starts with a comment

### ISSUE THREAD

**Issue by lacygoill** [vim/vim#6642]
**Describe the bug**

When a `:def` function raises an error, the address of the error line is wrong if the body of the function starts with commented lines.

**To Reproduce**

Run this shell command:

    vim -Nu NONE -S <(cat <<'EOF'
        vim9script
        def Func()
            # some
            # comment
            invalid
        enddef
        defcompile
    EOF
    )

`E476` is raised.

    Error detected while processing command line..script /proc/26833/fd/11[7]..function <SNR>1_Func:
    line    1:
    E476: Invalid command: invalid

This is expected.

The address of the error line is `1`:

    line    1:

This is not expected.

**Expected behavior**

The address of the error line is `3`, because `invalid` is on the third line in the body of `Func()`; it's not on the first line.

**Environment**

 - Vim version: 8.2 Included patches: 1-1382
 - OS: Ubuntu 16.04.6 LTS
 - Terminal: XTerm(358)

**Additional context**

Vim ignores the first commented lines inside a `:def` function; but not the other ones, which is inconsistent:
```vim
vim9script
def Func()
    echo 'valid'
    # some
    # comment
    invalid
enddef
defcompile
```
    Error detected while processing command line..script /proc/27050/fd/11[8]..function <SNR>1_Func:
    line    4:
    E476: Invalid command: invalid

This time, the address of the error line is `4` which is correct.  This means that the commented lines `# some` and `# comment` were not ignored when computing the address of the error line.

---

This breaks my [vim-stacktrace](https://github.com/lacygoill/vim-stacktrace) plugin, whose purpose is to parse the most recent errors and display them in the quickfix window, which makes it harder to debug an issue in a `:def` function.

**Comment by lacygoill** [comment]
I think the same issue applies to empty lines at the start of the function's body; they are ignored.  The other empty lines – in the middle of the body – are not ignored.

**Comment by lacygoill** [comment]
Fixed by [8.2.1392](https://github.com/vim/vim/commit/bf8feb5aeb71c8f56897845c4134793201aa459a).
